CHANGELOG.md

```diff
@@ -0,0 +1,268 @@
+    ```
+    
+- Add Registry/Config Access token capability via Secret Provider ([#229](https://github.com/edgexfoundry/app-service-configurable/issues/229)) ([#7b79720](https://github.com/edgexfoundry/app-service-configurable/commits/7b79720))
+    ```
+    BREAKING CHANGE:
+    When running with the secure Edgex Stack now need to have the SecretStore configured, and run with EDGEX_SECURITY_SECRET_STORE=true so a Vault token is created.
+    ```
+    
+- Enable Registry and Config Access Token ([#224](https://github.com/edgexfoundry/app-service-configurable/issues/224)) ([#4992cc8](https://github.com/edgexfoundry/app-service-configurable/commits/4992cc8))
+
+    ```
+    BREAKING CHANGE:
+    Service key changed to `app-<profile name>` to standerdize Vault token and Consul access token creation
+    ```
+
+- Add new FilterBySourceName to profiles ([#211](https://github.com/edgexfoundry/app-service-configurable/issues/211)) ([#02c1bdd](https://github.com/edgexfoundry/app-service-configurable/commits/02c1bdd))
+
+- Update profiles for latest SDK changes ([#196](https://github.com/edgexfoundry/app-service-configurable/issues/196)) ([#50a23fb](https://github.com/edgexfoundry/app-service-configurable/commits/50a23fb))
+    ```
+    BREAKING CHANGE:
+    requires SecretStore.Type be set to `vault`
+    ```
+    
+- Added new secrets configuration parameters to EncryptWithAES ([#193](https://github.com/edgexfoundry/app-service-configurable/issues/193)) ([#0f5331e](https://github.com/edgexfoundry/app-service-configurable/commits/0f5331e))
+
+- Update profiles for new filter enhancements ([#187](https://github.com/edgexfoundry/app-service-configurable/issues/187)) ([#8e9141f](https://github.com/edgexfoundry/app-service-configurable/commits/8e9141f))
+
+- Update profiles for new subscribe topic for V2 DTOs ([#175](https://github.com/edgexfoundry/app-service-configurable/issues/175)) ([#b2b5516](https://github.com/edgexfoundry/app-service-configurable/commits/b2b5516))
+
+- Update profiles for multiple MessageBus subscriptions ([#164](https://github.com/edgexfoundry/app-service-configurable/issues/164)) ([#2d8b7de](https://github.com/edgexfoundry/app-service-configurable/commits/2d8b7de))
+
+- Remove MarkAsPushed feature ([#159](https://github.com/edgexfoundry/app-service-configurable/issues/159)) ([#ed155f3](https://github.com/edgexfoundry/app-service-configurable/commits/ed155f3))
+
+    ```
+    BREAKING CHANGE:
+    MarkedAsPushed capability no longer supported
+    ```
+
+- Update configuration files to set default non-TLS ([#157](https://github.com/edgexfoundry/app-service-configurable/issues/157)) ([#0b6ac86](https://github.com/edgexfoundry/app-service-configurable/commits/0b6ac86))
+
+- Remove remote logging capability ([#143](https://github.com/edgexfoundry/app-service-configurable/issues/143)) ([#dc1b719](https://github.com/edgexfoundry/app-service-configurable/commits/dc1b719))
+### Bug Fixes 🐛
```

Comment:
Please insert new lines before and after title- easier to read the separation boundary and also lint from markdown

Reply:
Renders fine w/o extra lines

Reply:
ok, it is not the rendering issue; it is for editor's readability i guess. markdownlint flags me that warning.
it is like code compiling ok but having uneasy to read code- works ok but can be improved.

Reply:
I use Typora to edit. Much cleaner.

Reply:
I have typora too. it is just more convenient for me if all docs can be opened in one editor instead of two windows
PR request viewer editor doesn't have typora embedded unfortunately so a bit annoying when reads